### ISSUE TITLE
Ubuntu 18.04 ppa error

### ISSUE THREAD

**Issue by cemsbr** [pypa/pipenv#2380]
It would be nice adding to the installation instructions how to overcome the following error:
```
$ sudo add-apt-repository ppa:pypa/ppa
(...)
Err:11 http://ppa.launchpad.net/pypa/ppa/ubuntu bionic Release                                                                        
  404  Not Found [IP: 91.189.95.83 80]
Reading package lists... Done                             
E: The repository 'http://ppa.launchpad.net/pypa/ppa/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by cemsbr** [comment]
I tried ```deb [allow-insecure=yes] http://ppa.launchpad.net/pypa/ppa/ubuntu bionic main``` but pipenv cannot be found.

**Comment by uranusjr** [comment]
I don’t think Bionic is supported on the PPA at all. Why would you think it should work? (honest question, we should clean things up if they give wrong suggestions)

**Comment by cemsbr** [comment]
It's not written that bionic is supported, neither it's not (only 17.10 is cited). I thought that this was the case where users are using outdated instructions in more recent (Ubuntu) versions.

**Comment by acrogenesis** [comment]
Is there a way to install Pipenv on `18.04`? (I'm fairly new to Pipenv)

**Comment by uranusjr** [comment]
@acrogenesis I believe the most recommended way (by the community) is to use `pip3 install --user` and add `~/.local/bin` to the front of your `PATH`.

**Comment by kennethreitz** [comment]
ppa is not available for 18.04. we're removing support for the ppa, moving forward.
